src/components/TraceQueryBrowser.tsx

```diff
@@ -0,0 +1,214 @@
+import React from 'react';
+
+import {
+  ChartsProvider,
+  generateChartsTheme,
+  getTheme,
+  SnackbarProvider,
+} from '@perses-dev/components';
+import {
+  DataQueriesProvider,
+  dynamicImportPluginLoader,
+  PluginModuleResource,
+  PluginRegistry,
+  TimeRangeProvider,
+} from '@perses-dev/plugin-system';
+import { ThemeProvider } from '@mui/material';
+import { QueryClient, QueryClientProvider } from '@tanstack/react-query';
+import {
+  DatasourceStoreProvider,
+  TemplateVariableProvider,
+} from '@perses-dev/dashboards';
+import panelsResource from '@perses-dev/panels-plugin/plugin.json';
+import {
+  DashboardResource,
+  GlobalDatasource,
+  ProjectDatasource,
+} from '@perses-dev/core';
+import { DatasourceApi } from '@perses-dev/dashboards';
+import tempoResource from '@perses-dev/tempo-plugin/plugin.json';
+import { TextInput, Button } from '@patternfly/react-core';
+import { TraceTable } from './TraceTable';
+import { PersesChartsTheme } from '@perses-dev/components';
+import { Stack, StackItem } from '@patternfly/react-core';
+import { ScatterChart } from '@perses-dev/panels-plugin';
+
+import './TraceQueryBrowser.css';
+
+// To configure a different endpoint for your Tempo instance
+// you need to modify the env variable 'BRIDGE_PLUGIN_PROXY'
+// in the start-console.sh script.
+const proxyDatasource: GlobalDatasource = {
```

Comment:
it will be good to extract all this perses configuration into its own file

Reply:
Good point.